### ISSUE TITLE
support for Model.create_table() to create a table with default autoscaling capacity

### ISSUE THREAD

**Issue by devangmehta123** [pynamodb/PynamoDB#417]
I have tried Model.create_table() in version 3.2 without setting read_capacity_units and write_capacity_units. pynamo throws an exception that saying that it needs ints between 1-inf for these.

But I was hoping that it would create the tables with the default AWS autoscale settings which apply when you manually create the table through the console.

And to me it seems to be a very useful feature addition which could hlep many users of pynamodb.

If there is a workaround, please let me know.

**Comment by dmulter** [comment]
I'm actually just looking for support of the new auto-scaling feature. I assume there would be new Meta fields for controlling read/write scaling and the associated parameters.

**Comment by kanendosei** [comment]
Any update on this issue? I would very much like to create a table from pynamodb with autoscaling enabled by default.

**Comment by dhruvsakalley** [comment]
It'll be interesting to be able to manage dynamodb scaling even post creation. I'm ingesting large data files into dynamodb and then reducing capacity to apply autoscaling later. I wonder how the update scenarios will be handled here.

**Comment by andreborges-janrain** [comment]
Any news about that?

**Comment by Seluj78** [comment]
Ping ?

**Comment by ikonst** [comment]
I think right now our story for provisioning and reprovisioning tables on actual AWS DynamoDB is pretty weak. We're using PynamoDB at Lyft, and within Lyft we effectively only use PynamoDB's `create_table` in development against dynamodb-local and similar databases, while we do our provisioning with Terraform.

**Comment by Seluj78** [comment]
Well same here @ikonst, just that I was wondering if it would be possible to have an option in `create_table` to set the auto scaling parametters like in the console

<img width="772" alt="Screenshot 2019-08-14 at 20 33 34" src="https://user-images.githubusercontent.com/4641317/63046484-ce982d80-bed2-11e9-82b0-bb5c36803a1c.png">

**Comment by ikonst** [comment]
@Seluj78 What I'm asking is how is it useful to you? Are you provisioning **production** tables with PynamoDB?

**Comment by Seluj78** [comment]
@ikonst this is totally going to look dumb but I forgot why I needed it... So nvm I guess, my bad 😖

**Comment by Seluj78** [comment]
But no I am not

**Comment by rblakemesser** [comment]
I'd definitely like to be able to provision production tables this way fwiw

**Comment by monkut** [comment]
Yes, if we have `read_capacity_units` and `write_capacity_units` settings in Meta, it would be helpful to have auto-scale capability as well.

```
    class Meta:
        read_capacity_units = 2  # with auto-scale use as minimum?
        max_read_capacity_units = 10
        write_capacity_units = 2  # with auto-scale use as minimum?
        max_write_capacity_units = 10  
```

Is there any down side to setting the auto-scale configuration? 
If not, maybe default `max_read_capacity_units` to `read_capacity_units`, and if given use that value?
